.github/workflows/release.yaml

```
@@ -45,13 +45,27 @@ jobs:
                   npx lerna version ${{ github.event.inputs.release_type }} --conventional-commits --yes --no-private --force-publish
                   npx lerna publish from-git --yes --dist-tag ${{ github.event.inputs.release_type == 'preminor' && 'next' || 'latest' }}
 
+            - name: Get Version Tag
+              id: get_tag
+              run: echo "TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:49: Double quote to prevent globbing and word splitting

(shellcheck)